ports/asiochan/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "asiochan",
+  "version": "0.4.3",
```

```suggestion
  "version-date": "2022-11-25",
```
Since there is no tag or release could be used in https://github.com/MiSo1289/asiochan, so we should use the date of the commit as the port version.

---

The project use conan and really has a version [here](https://github.com/MiSo1289/asiochan/blob/837d7eb78ca9796af800ca3cd91ce0a8fe297785/CMakeLists.txt#L2)

---

In vcpkg, the value of `version` is typically a version number explicitly mentioned on the port's website, such as the release or tag values on GitHub. If a port on GitHub does not have any releases or tags, a common approach is to use its latest commit. In this case, we use the `version-date` field in the `vcpkg.json` file, and its value is the generation date of the used commit.

---

Since you insist, I'll change it. However, I am not the author, I'm not sure this offends the author.

---

It is possible to ask the author to tag releases.
The version in CMakeLists.txt isn't enough to identify the release commit: Some project bump it immediately before a release, other immediately after a release.

---

But vcpkg reference the project using commit hash, not version. If the author update the project without change version, it means that he think the change not impact the consumers of the library. The last commit is at two years ago, So I think the version is fixed.